### DISCUSSION TITLE
When is the next release for snakeYAML 2.2

### DISCUSSION THREAD

**Discussion by Hans592** [Netcentric/accesscontroltool#680] [Q&A] [answered]
Hello,

I am currently using this repo in my project. However, there is a security vulnerability with snakeYAML version 1.32 https://mvnrepository.com/artifact/org.yaml/snakeyaml/1.32. I can see in the develop branch that there is an updated snakeYAML in the pom.xml https://github.com/Netcentric/accesscontroltool/blob/develop/pom.xml#L118.

My question : when is the next release that will have an updated snakeYAML to version 2.2?

Thanks,
Steven

**Comment by kwin** [comment, marked as answer]
https://github.com/Netcentric/accesscontroltool/releases/tag/3.0.10
